### PR TITLE
feat(cart): builder io - drag drop area above and below cart template…

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**
… content

### CODE REVIEW

File: src/pages/cart.tsx [hunk 1]
```diff
@@ -1,25 +1,124 @@
+import { BuilderComponent, builder, Builder } from '@builder.io/react'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import getConfig from 'next/config'
 
+import SmallBanner from '@/components/home/SmallBanner/SmallBanner'
```
Comment (TriptiVermaXP) [inline src/pages/cart.tsx]: use index.ts for import export

**Review by TriptiVermaXP — APPROVED** [review]
Please check minor comment.

File: src/pages/cart.tsx (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply src/pages/cart.tsx]: Resolved

**Review by kibo-sushant — APPROVED** [review]

File: src/pages/cart.tsx [hunk 2]
```diff
@@ -1,25 +1,124 @@
...
+  ],
+})
+
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const { locale, req, res } = context
   const response = await getCart(req, res)
+
+  const bannerSection = await builder.get('cart-banner-section').promise()
+  const cartProductAndCategorySection = await builder
+    .get('cart-product-and-category-section')
```
Comment (kibo-kevinwatts) [inline src/pages/cart.tsx]: can we rename the sections to be less "type" specific and more location specific?
like `cart-top-content-section` / `cart-bottom-content-section` or something.
Update the CartPage/Template props to something similar too

Using builder, we can drag other blocks and components in beside "product and category"  related items.

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: src/pages/cart.tsx (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply src/pages/cart.tsx]: Resolved.
Yes, we can drag other blocks and components also.

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**